packages/next-on-pages/templates/_worker.js/index.ts

```diff
@@ -25,8 +26,9 @@ export default {
 				{ status: 503 },
 			);
 		}
+
 		return envAsyncLocalStorage.run(
-			{ ...env, NODE_ENV: __NODE_ENV__ },
+			{ ...env, NODE_ENV: __NODE_ENV__, SUSPENSE_CACHE_URL },
```

Comment:
Because that's the env variable used in the fetch cache so that it knows where to send the api requests for the cache. those api requests are the ones we then intercept.

Reply:
ahhhh, ok it's Next that it's accessing it I see! (https://github.com/vercel/next.js/blob/2cf5d3a8aa325f8bece1094c9e566311e604e114/packages/next/src/server/lib/incremental-cache/fetch-cache.ts#L28)

Could you add a comment regarding that so that we're sure to remember that it's used by them and not us? (otherwise this could be removed as it doesn't look like actually being used)